src/facade/reply_builder.cc

```diff
@@ -593,47 +593,54 @@ void RedisReplyBuilder::SendStringArrInternal(
   serialize_len(type_char[0], header_len);
   unsigned vec_indx = 0;
   string_view src;
+
+#define FLUSH_IOVEC()           \
```

Comment:
Why is this a macro and not a lambda?

Reply:
because it returns from the parent scope

Reply:
sure and in that case you just write:

```
      if (vec_indx >= vec_cap && FLUSH_IOVEC()) {
        return;
      }
```

Which IMO is almost the same. It's a nit, don't pay much attention to it if you don't want to :)